plugins/module_utils/foreman_helper.py

```diff
@@ -1541,6 +1540,10 @@ def _recursive_dict_without_none(a_dict, exclude=None):
     return result
 
 
+def _is_resolved(what):
+    return (what is None or isinstance(what, (dict, list)))
```

Comment:
Right, you can't check if it's a list, when it's already a list of strings. If it's a list, the elements need to be dicts.

Reply:
That means the test in the nested thing is not entirely correct, but it didn't trigger because we didn't have entity_lists there?
If you passed the spec, you could make the test more specific.

Reply:
Exactly.

Reply:
How about that.